app/admin/facilitator_guide.rb

```diff
@@ -17,7 +16,7 @@
     f.inputs "Facilitator Guide" do
       f.input :title
       f.input :level
-      f.input :session
+      f.input :session, input_html: { class: "selectpicker" }
```

Comment:
Boss @andela-mpitan. We need to nest the creation of facilitator guide under sessions.
`sessions/:session_id/facilitator_guide/new`
So we can easily use the session id to find session and create the facilitator_guide using the has_many relationship a session has with a facilitator_guide.

Reply:
Are you suggesting we use nested forms in the sessions view

Reply:
Oh! I see what you are saying now. `ActiveAdmin` already handles it for us. You just need to declare your relationships in the model.